src/types/ut_types/UtilsTests.cpp

```diff
@@ -237,6 +242,12 @@ void UtilsTests::TestColorFromXTermColor()
     _VerifyXTermColorInvalid(L"rgbİ:1/1/1");
     _VerifyXTermColorInvalid(L"rgß:1/1/1");
     _VerifyXTermColorInvalid(L"rgẞ:1/1/1");
+    _VerifyXTermColorInvalid(L"yellow3a");
+    _VerifyXTermColorInvalid(L"3yellow");
+    _VerifyXTermColorInvalid(L"royal3blue");

```

Comment:
You know why I don't use "yello3w"? Because spell check complains about "yello" is not a word...Thank god we have "royal blue"